tests/test_validate_outputs.py

```diff
@@ -77,26 +77,26 @@ def test_calexp(self):
         # running the test to determine the updated values.
         expected_places = 6
```

Comment:
Why do these values need to change by ~1e-15 then?

Reply:
Hmm. I see the summary RA/dec changed by ~1e-9.

If they're all just noise I wouldn't change them; if some are actually marginally significant then okay, may as well keep them all.